### PR TITLE
GAN: generator logits should not apply activation

### PR DESCRIPTION
**Pull request by Piasy**

### CODE REVIEW

**Comment by mcleonard** [comment]
The activation for tf.layers.dense is set to None by default, so we don't need to specify it here. However, making it explicit is better, so I'll merge this.
